src/views/profile/edit.ejs

```diff
@@ -114,19 +114,19 @@
         <option value="CL">Chile</option>
         <option value="CN">China</option>
         <option value="CX">Christmas Island</option>
-        <option value="CC">Cocos (Keeling) Islands</option>
+        <option value="CC">Cocos Islands</option>
         <option value="CO">Colombia</option>
         <option value="KM">Comoros</option>
         <option value="CG">Congo</option>
-        <option value="CD">Congo, the Democratic Republic of the</option>
         <option value="CK">Cook Islands</option>
         <option value="CR">Costa Rica</option>
         <option value="CI">Côte d'Ivoire</option>
         <option value="HR">Croatia</option>
         <option value="CU">Cuba</option>
         <option value="CW">Curaçao</option>
         <option value="CY">Cyprus</option>
-        <option value="CZ">Czech Republic</option>
+        <option value="CZ">Czechia</option>
```

Comment:
Is Czech Republic more official? 
https://en.wikipedia.org/wiki/Czech_Republic

Reply:
Both are acceptable, but I believe ISO has it listed as Czechia

Reply:
Hmm, could we keep it as Czech Republic then? Just anecdotally that's what I've been more familiar with. But if you think Czechia is more common/official let's go with that. I'll leave it up to you :).